hmda/src/main/scala/hmda/api/http/ParserErrorUtils.scala

```diff
@@ -32,7 +33,7 @@ object ParserErrorUtils {
           FieldParserErrorSummary(
             errorMessage.fieldName,
             errorMessage.inputValue,
-            lookupParserValidValues(errorMessage.fieldName)
+            lookupParserValidValuesByYear(errorMessage.fieldName,period)
```

Comment:
@lchen-2101  came up with a solution to reference an alternative valid value conf file.  let me know if this is too wild of an approach